index.html

```diff
@@ -0,0 +1,136 @@
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
+    </div>
+
+    </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/Portfolio.png" alt="">
+        </div>
+        <div class="card-header">
+          <h3>Multi-post <br> stories</h3>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Frontend Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2022</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">HTML</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">CSS</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">Javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+            </div>
+      </section>
+      <!--end of work section-->
+
+      <!--end about me section-->
+      <section class="about_me">
+        <div class="about_container">
+          <h3 class="about_me_title">About<br>Myself</h3>
+          <div class="about_info">
+            <h3 class="about_me_title-desktop">About me</h3>
+            <div class="about_description">
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don’t
+                hestiate to contact me.
+              </p>
+            </div>
+            <div>
+              <button type="button" class="about_btn">Get My Resume</button>
+            </div>
+          </div>
+          <div class="about-language">
+            <div class="lang-title">
+              <h2>Languages</h2>
```

Comment:
Kindly this title miss an arrow down icon and try to reduce the title is too big differently to the Figma template

Reply:
<div class="lang-title">
              <h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>
            </div>

Reply:
<h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>